security-policies/bundle/compliance/cis_azure/rules/cis_1_23/test.rego

```diff
@@ -0,0 +1,75 @@
+package compliance.cis_azure.rules.cis_1_23
+
+import data.cis_azure.test_data
+import data.compliance.policy.azure.data_adapter
+import data.lib.test
+
+test_violation {
+	eval_fail with input as test_data.generate_azure_asset("azure-role-definition", {
+		"assignableScopes": [
+			"/",
+			"/subscriptions/11111111-2222-3333-4444-555555555555",
+		],
+		"permissions": [{"actions": ["*"]}],
+		"type": "CustomRole",
+	})
+	eval_fail with input as test_data.generate_azure_asset("azure-role-definition", {
+		"assignableScopes": ["/subscriptions/11111111-2222-3333-4444-555555555555"],
+		"permissions": [{"actions": ["*"]}],
+		"type": "CustomRole",
+	})
+	eval_fail with input as test_data.generate_azure_asset("azure-role-definition", {
+		"assignableScopes": ["/"],
+		"permissions": [{"actions": ["*"]}],
+		"type": "CustomRole",
+	})
+	eval_fail with input as test_data.generate_azure_asset("azure-role-definition", {
+		"assignableScopes": [
+			"RandomScope",
+			"/",
+			"/subscriptions/11111111-2222-3333-4444-555555555555",
+		],
+		"permissions": [{"actions": ["RandomAction", "*"]}],
+		"type": "CustomRole",
+	})
+}
+
+test_pass {
+	eval_pass with input as test_data.generate_azure_asset("azure-role-definition", {
+		"assignableScopes": [
+			"/",
+			"/subscriptions/11111111-2222-3333-4444-555555555555",
```

Comment:
Can you also add a pass with scope set to `/subscriptions/11111111-2222-3333-4444-555555555555/some-path`?